core/src/bundle_stage.rs

```diff
@@ -304,24 +304,25 @@ impl BundleStage {
             &bank_start.working_bank,
         );
 
-        // qos rate-limited a tx in here, drop the bundle
+        // accumulates QoS to metrics
+        qos_service.accumulate_estimated_transaction_costs(
```

Comment:
i think we might wanna move this to after the `if sanitized_bundle.transactions.len() != num_included`?

Reply:
to where its at right now

Reply:
you mean inside the if block?  

if you put it after the if block (like it was before) it doesn't get hit for cost model erros

Reply:
hmm i see.. problem here is that it would overcount metrics if the if statement is hit

Reply:
however looking at the metrics for banking stage it looks like it could do something simlar

Reply:
i want this code to be hit, so i can look for these on bundle stage thread id
https://github.com/jito-foundation/jito-solana/blob/30b47b8ff92081a0b8ee9382ce08baeae4a0cffe/core/src/bundle_stage.rs#L235-L243

Reply:
makes sense... i think banking_stage has the same issue, so probably ok

Reply:
update the estimated: https://github.com/jito-foundation/jito-solana/blob/master/core/src/banking_stage.rs#L1757

here all txs might not be free to lock, which means there might be double counting for estimated cus: https://github.com/jito-foundation/jito-solana/blob/master/core/src/banking_stage.rs#L1774